mmdeploy/pytorch/functions/cat.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Sequence
+
+import torch
+from torch import Tensor
+
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import get_dynamic_axes
+
+
+@FUNCTION_REWRITER.register_rewriter(func_name='torch.cat', backend='tensorrt')
+def cat__tensorrt(tensors: Sequence[Tensor], *args, **kwargs) -> torch.Tensor:
+    """Rewrite `cat` for TensorRT backend.
+
+    cat in TensorRT does not support bool or uint8 type when input is dynamic.
+    """
+    ctx = FUNCTION_REWRITER.get_context()
+    if get_dynamic_axes(ctx.cfg) is None:
```

Comment:
warning: there is a potential risk. deploy config is not required for tensorrt.from_onnx.

Reply:
I suppose it only works for `torch2onnx` with `dynamic_axies` not None.

Reply:
@grimoire can we handle this case in the latest refactor plan?

Reply:
@lvhan028 Nope